scripts/nx-release.ts

```diff
@@ -1,14 +1,14 @@
 #!/usr/bin/env node
-import * as yargs from 'yargs';
 import { execSync } from 'child_process';
 import { existsSync, readFileSync, writeFileSync } from 'fs';
-import { URL } from 'url';
 import { join } from 'path';
+import { URL } from 'url';
+import * as yargs from 'yargs';
 
 import { parse } from 'semver';
 
-import * as version from '@lerna/version/index';
-import * as publish from '@lerna/publish/index';
```

Comment:
Any idea why this had to change?

Reply:
Yes, the commands are no longer exported from separate packages and are instead exported by `lerna`. We don't really encourage direct import of the functions like this anyway so the typescript types aren't exported either.